### DISCUSSION TITLE
How can I edit the theme

### DISCUSSION THREAD

**Discussion by Niblick1020** [ubai-m/react-portfolio#51] [General]
Thank you so much for posting this to GitHub and allowing us to use it.

If we want to edit the theme to change the format or photos how could we go about this? Would we need to use react?

**Comment by ubaimutl** [comment]
Hi Niblick1020,

To change the photo and theme colors, you'll need to edit two specific files:

1. **Change the Photo:**
   - Go to `src/content_option.js`.
   - Locate the `introdata` object.
   - Change the `your_img_url` value to the URL of your photo.

2. **Change the Theme Color:**
   - Open `src/index.css`.
   - Here, you will find two sections:
     - `:root` for the default (dark) theme.
     - `[data-theme="light"]` for the light theme.
   - Edit the color variables (`--bg-color`, `--primary-color`, etc.) to your preferred colors.

Hope this helps!

**Reply by Niblick1020** [reply]
It did. thank you!
